corehq/ex-submodules/couchforms/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+from django.conf import settings
```

Comment:
this is pretty tricky. in order to get django to register your models when they're in a folder you need to import them into the `__init__.py` file. at first i thought it'd be cool to dynamically use couch or postgres models based on the configuration in the settings file. now i'm thinking it makes most sense to make all models available at `couchforms.models.<couch|sql>.Model`